<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitHubCreatePullRequestPopover</class>
 <widget class="QWidget" name="GitHubCreatePullRequestPopover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="initialPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="tTitleLabel" name="titleLabel">
         <property name="text">
          <string>Create Pull Request</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="prOptionsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1">
           <widget class="QLineEdit" name="titleEdit"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="tSubtitleLabel" name="label">
            <property name="text">
             <string>Pull Request Options</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" alignment="Qt::AlignTop">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Comment</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPlainTextEdit" name="commentEdit"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string notr="true">TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="tStatusFrame" name="prFailedFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="createPrButton">
         <property name="text">
          <string>Create Pull Request</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadingPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>134</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>273</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="tCircularSpinner" name="widget" native="true"/>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>273</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>134</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tCircularSpinner</class>
   <extends>QWidget</extends>
   <header location="global">tcircularspinner.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
   <slots>
    <signal>switchingFrame(int)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tSubtitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">tsubtitlelabel.h</header>
  </customwidget>
  <customwidget>
   <class>tStatusFrame</class>
   <extends>QFrame</extends>
   <header location="global">tstatusframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
